src/Persistence.php

```diff
@@ -45,20 +45,11 @@ public static function connect($dsn, string $user = null, string $password = nul
         $dsn = \Atk4\Data\Persistence\Sql\Connection::normalizeDsn($dsn, $user, $password);
 
         switch ($dsn['driverSchema']) {
+            case 'sqlite':
             case 'mysql':
-            case 'oci':
-            case 'oci12':
```

Comment:
For some reason which I don't remember now we had separated Oracle 12 support.

Reply:
There was a different compatibility layer previously for Oracle 12, but I purged some time ago it in favor of generic layer for Oracle 11 and all tests are run againt this version in CI.